### PR TITLE
Add a notifier for Librato

### PR DESCRIPTION
**Pull request by jjbohn**
So we can annotate the graphs with deployments

### CODE REVIEW

File: lib/heaven/notifier.rb [hunk 1]
```diff
@@ -3,22 +3,15 @@
...
-        ::Heaven::Notifier::Hipchat.new(payload)
-      elsif flowdock?
-        ::Heaven::Notifier::Flowdock.new(payload)
-      elsif Rails.env.test?
-        # noop on posting
-      else
-        ::Heaven::Notifier::Campfire.new(payload)
-      end
+      %i(flowdock librato slack hipchat campfire).map { |notifier|
+        "::Heaven::Notifier::#{notifier.capitalize}".constantize.new(payload) if send("#{notifier}?")
```
Comment (daemonsy) [inline lib/heaven/notifier.rb:13]: should probably send a PR to Heaven for this.much cleaner :smile:

File: lib/heaven/notifier/librato.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "#{chat_user}'s deployment of #{repo_name} (#{sha}) #{message_completion_map[state]}."
+      end
+
+      private
+
+      def librato_client
+        @librato_client ||= ::Librato::Metrics.authenticate(ENV["LIBRATO_EMAIL"], ENV["LIBRATO_API_KEY"])
+      end
+
+      def commitish_url
```
Comment (daemonsy) [inline lib/heaven/notifier/librato.rb:36]: What does commitish mean?

Comment (jjbohn) [inline lib/heaven/notifier/librato.rb:36]: Commitish is git's name for a `commit`, `tag`, or `branch`.

Comment (jjbohn) [inline lib/heaven/notifier/librato.rb:36]: Anything that you can `checkout` to basically.

**Comment by daemonsy** [comment]
:+1:

**Pull request merged**